### PR TITLE
Getting 404 url while updating quantity on multiple address cart page

### PR DESCRIPTION
**Pull request by vikalps4**
Getting 404 form validation url while updating quantity on multiple address cart page

### Description (*)
Getting 404 page not found form validation url while updating item quantity on multiple address cart page.

### Fixed Issues (if relevant)
1. Resolved 404 form validation url while updating item quantity on multiple address cart page.

### Manual testing scenarios (*)
1. Add product to cart
2. Go to cart page
3) Checkout with multiple addresses.
4) Update quantity and click "Update Qty & Addresses"

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=22717) <br/>All committers have signed the CLA.

**Comment by ronak2ram** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ronak2ram. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @ronak2ram, here is your [Magento instance](https://i-22717-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-22717-2-3-develop.instances.magento-community.engineering/admin](https://i-22717-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by ronak2ram** [comment]
Hi @vikalps4 
Thanks for your contributions.
I checked in with Magento 2.3-develop instance and not getting an issue.
Can you please verify with 2.3-develop instance.

**Comment by vikalps4** [comment]
Hi @ronak2ram 
Please follow the following steps :
1. Add product to cart.
2. Checkout with multiple addresses.
3. Edit any item quantity and add random quantity(more than available stock like 343343) and click "Update Qty & Addresses" button.
4. Cart form validation request goes to 404 URL. 

Frontend validation is not working due to validation request going to 404 URL.

**Comment by vikalps4** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @vikalps4. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by vikalps4** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @vikalps4. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @vikalps4, here is your [Magento instance](https://i-22717-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-22717-2-3-develop.instances.magento-community.engineering/admin](https://i-22717-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/addresses.phtml [hunk 1]
```diff
@@ -18,7 +18,7 @@
       data-mage-init='{
           "multiShipping":{},
           "cartUpdate": {
-               "validationURL": "/multishipping/checkout/checkItems",
+               "validationURL": "<?= /* @escapeNotVerified */ $block->getUrl('multishipping/checkout/checkItems') ?>",
```
Comment (sidolov) [inline app/code/Magento/Multishipping/view/frontend/templates/checkout/addresses.phtml]: Please, avoid usage `escapeNotVerified` in the new or changed code, it's better to escape current string

**Review by sidolov — CHANGES_REQUESTED** [review]

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/addresses.phtml (hunk 1 above)
Comment (vikalps4) [reply app/code/Magento/Multishipping/view/frontend/templates/checkout/addresses.phtml]: Hi @sidolov 
I have updated the change, please review it.

Comment (sidolov) [reply app/code/Magento/Multishipping/view/frontend/templates/checkout/addresses.phtml]: Perfect! Thanks!

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5370](https://jira.corp.magento.com/browse/ENGCOM-5370) has been created to process this Pull Request
:eight_spoked_asterisk: @sidolov, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
Hi @vikalps4 unfortunately, I cannot reproduce issue described by PR on 2.3-develop.
**Manual testing scenario:**

- Register customer and sign in
- Add product to cart.
- Go to cart page 
- Click Check Out with Multiple Addresses.
- Edit any item quantity and add random quantity(more than available stock like 343343) and click "Update Qty & Addresses" button.

**Result:**
Attention popup with validation message is shown
#22717MainNotrepro


Could you take a look?

**Comment by vikalps4** [comment]
Hi @engcom-Delta 
Yes, now the attention popup with validation message is coming fine, may be it is fixed now.
However, i believe we should not use this hardcoded URL value 

> "validationURL": "/multishipping/checkout/checkItems",

**Pull request merged**